interface/modules/custom_modules/oe-module-claimrev-connect/public/setup.php

```diff
@@ -0,0 +1,165 @@
+<?php
+    namespace OpenEMR\Modules\ClaimRevConnector;
+    $tab="setup";
+    require_once '../src/ClaimRevModuleSetup.php';
```

Comment:
remove above line and instead require_once globals.php to both ensure access to authenticated users only and ensure your classes are autoloaded